introduction.md

```diff
@@ -1,2 +1,17 @@
 I'm Genelle Wertz, I'm here to calm the deadly vipers.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?